webui/react/src/components/LogViewerStore.ts

```diff
@@ -0,0 +1,40 @@
+import { TrialLog } from '../types';
```

Comment:
**nit:** '/types' without '..'

Reply:
**suggestion:** how about calling this file `LogViewer.store.ts`? To follow the pattern of a the component that is supports (LogViewer), followed by it's function (store). Similar to `X.table.tsx` (although it should be `ts` instead of `tsx` for tables)